extensions/stackdriver/log/logger.cc

```diff
@@ -50,13 +50,17 @@ using google::protobuf::util::TimeUtil;
 
 // Name of the HTTP server access log.
 constexpr char kServerAccessLogName[] = "server-accesslog-stackdriver";
+constexpr char kAuditAccessLogName[] = "server-accesslog-stackdriver-audit";
```

Comment:
Out of curiosity, have you thought about fill in `protoPayload` in logEntry, which supports `google.cloud.audit.AuditLog`: https://cloud.google.com/logging/docs/reference/audit/auditlog/rest/Shared.Types/AuditLog. Sound like that is a more standard way to do audit log at Google?

Reply:
Yes, the AuditLog payload is used primarily for auditting GCP service calls, it doesn't apply to application logging, afaik.

Reply:
@bianpengyuan @davidraskin can you find out if `google.cloud.audit.AuditLog` **cannot** be used to audit other services?

Reply:
@mandarjog Maybe it can theoretically be used, because I think the only difference is the proto payload, but the [Audit Log docs](https://cloud.google.com/logging/docs/reference/audit/auditlog/rest/Shared.Types/AuditLog#AuthenticationInfo) state that it is for GCP API calls. Additionally, [configuring the Data Access Audit Logs](https://cloud.google.com/logging/docs/audit/configure-data-access) is done through the Cloud Console, which could be confusing since that wouldn't be the case here.

Reply:
Should these logs be actually called audit logs? https://cloud.google.com/logging/docs/audit cloud audit logs looks like are logs for telling who or what accessed a GCP resource... This is more about capturing auth info for application?

Reply:
They're still audit logs I think. Just ones at the application level, separate from GCP audit logs. We can call them IstioAudit logs to reduce confusion.